packages/auth/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@unstoppabledomains/firebase-auth",
-  "version": "1.5.6",
+  "version": "1.5.7",
```

Comment:
Per Slack, we need a change in the `app` repo which we are not using a forked version. It might be a bit extra work.